typescript/generate.ts

```diff
@@ -113,6 +114,10 @@ async function main() {
   // copy readme and license
   await copyFile("../LICENSE", "packages/dbmate/LICENSE");
   await copyFile("../README.md", "packages/dbmate/README.md");
+
+  await exec("npm", ["run", "build"], {
+    cwd: "packages/dbmate",
+  });
```

Comment:
Another location for this could be the CI steps rather than here in `generate.ts`